tests/test_features_feature_collection.py

```diff
@@ -410,6 +388,49 @@ def quantiles_abs_diff(
     assert (res_df[expected_output_names[0]] != res_df[expected_output_names[2]]).any()
 
 
+def test_series_funcs(dummy_data):
+    def min_max_time_diff(x: pd.Series, mult=1):
+        diff = (x.index[1:] - x.index[:-1]).total_seconds().max()
+        return diff.min()*mult, diff.max()*mult
+    def time_diff(x: pd.Series):
+        return (x.index[-1] - x.index[0]).total_seconds()
+
+    fc = FeatureCollection(
+        MultipleFeatureDescriptors(
+            functions=[np.mean, np.sum, len, 
+                FuncWrapper(
+                    min_max_time_diff, input_type=pd.Series, 
+                    output_names=["min_time_diff", "max_time_diff"], mult=3,
+                ),
+                FuncWrapper(time_diff, input_type=pd.Series),
+                FuncWrapper(np.max, input_type=pd.Series)
```

Comment:
Waarom hier opnieuw een `pd.Series` als input-type?
-> lijkt mij interessant om een featurecollection te hebben met mixed input types

Reply:
Dat is toch het geval met np.mean, np.sum en len (lijn 400)